standard/lpdb/count.lua

```diff
@@ -0,0 +1,174 @@
+---
+-- @Liquipedia
+-- wiki=commons
+-- page=Module:Count
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local Class = require('Module:Class')
+local Logic = require('Module:Logic')
+local String = require('Module:StringUtils')
+
+local Condition = require('Module:Condition')
+local ConditionTree = Condition.Tree
+local ConditionNode = Condition.Node
+local Comparator = Condition.Comparator
+local BooleanOperator = Condition.BooleanOperator
+local ColumnName = Condition.ColumnName
+
+local Count = {}
+
+
+function Count.games(args)
+	args = args or {}
+
+	local lpdbConditions = Count._baseConditions(args)
+
+	return Count._query('game', 'objectname', lpdbConditions)
+end
+
+function Count.matches(args)
+	args = args or {}
+
+	local lpdbConditions = Count._baseConditions(args)
+	lpdbConditions = Count._tierConditions(args, lpdbConditions)
+
+	return Count._query('match', 'objectname', lpdbConditions)
+end
+
+function Count.tournaments(args)
+	args = args or {}
+
+	local lpdbConditions = Count._baseConditions(args, true)
+	lpdbConditions = Count._tierConditions(args, lpdbConditions)
+
+	return Count._query('tournament', 'objectname', lpdbConditions)
+end
+
+function Count.placements(args)
+	args = args or {}
+
+	local lpdbConditions = Count._baseConditions(args)
+	local player = mw.ext.TeamLiquidIntegration.resolve_redirect(args.player) or ''
```

Comment:
- will error for empty args.player
- the ` or ''` is redundant

Reply:
Oh yes, left this here to say: - the old module only queried counting player placement appearances, but I can think of uses for using it for team so I will clean the local player to be `opponent` and select accordingly